falcon/response.py

```diff
@@ -218,7 +218,13 @@ def set_cookie(self, name, value, expires=None, max_age=None,
                 self._cookies[name]["expires"] = gmt_expires.strftime(fmt)
 
         if max_age:
-            self._cookies[name]["max-age"] = max_age
+            # RFC 6265 section 5.2.2 says about the max-age value:
+            #   "If the remainder of attribute-value contains a non-DIGIT
+            #    character, ignore the cookie-av."
+            # That is, RFC-compliant response parsers will ignore the max-age
+            # attribute if the value contains a dot, as in floating point
+            # numbers. Therefore, attempt to convert the value to an integer.
+            self._cookies[name]["max-age"] = int(max_age)
```

Comment:
Could you also add a note to the `set_cookie()` docstring, mentioning that a float will be coerced to an int? You might also mention that a string value is acceptable (e.g., `'60'` vs. `60`).